octohook/events.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, payload):
         self.installation = Installation(payload.get("installation"))
 
         self.repositories = [
-            ShortRepository(repo) for repo in payload.get("repositories")
+            ShortRepository(repo) for repo in payload.get("repositories", list())
```

Comment:
```suggestion
            ShortRepository(repo) for repo in payload.get("repositories", [])
```

`[]` is a _little_ faster. https://stackoverflow.com/a/5790954